node/pkg/driver/device_connectivity/device_connectivity_helper_scsigeneric.go

```diff
@@ -505,48 +587,58 @@ func (o OsDeviceConnectivityHelperGeneric) ReloadMultipath() error {
 	return nil
 }
 
-func (o OsDeviceConnectivityHelperGeneric) GetDmsPath(volumeId string, volumeNguid string) (string, error) {
-	volumeUuidLower := strings.ToLower(volumeId)
-
-	mpathdOutput, err := o.Helper.WaitForDmToExist(volumeUuidLower, volumeNguid, WaitForMpathRetries, WaitForMpathWaitIntervalSec)
-
+func (o OsDeviceConnectivityHelperGeneric) GetMpathDeviceName(volumePath string) (string, error) {
+	procMountsFileContent, err := ioutil.ReadFile(procMountsFilePath)
```

Comment:
```suggestion
	procMountsFileContent, err := ioutil.ReadFile(procMountsFilePath)
	procMounts := string(procMountsFileContent)
```
since we only use it as `string`, we should convert it as soon as possible

Reply:
done